planet_scope/ndci/eob.js

```diff
@@ -0,0 +1,58 @@
+//VERSION=3
+//PlanetScope NDCI EO Browser
+
+function setup() {
+  return {
+    input: [{
+      bands: [
+        "red",
+        "rededge",
+        "green",
+        "blue",
+        "dataMask",
+        "clear"
+      ]
+    }],
+    output: [
+      { id: "default", bands: 4 },
+      { id: "index", bands: 1, sampleType: "FLOAT32" },
+      { id: "eobrowserStats", bands: 2, sampleType: "FLOAT32" },
+      { id: "dataMask", bands: 1 },
+    ]
+  };
+}
+
+function evaluatePixel(sample) {
+  let band1 = sample.rededge;
+  let band2 = sample.red;
+  const denominator = band1 + band2;
+  let ndci = denominator === 0 ? NaN : (band1 - band2) / denominator;
+
+  const clear = (sample.dataMask * sample.clear);
```

Comment:
functionally exactly the same

```suggestion
  const clear = sample.dataMask && sample.clear;
```